curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f42a021625f656101ef93.md

```diff
@@ -28,7 +28,7 @@ assert(document.querySelector('span')?.classList?.contains('right'));
 Your `span` element should have the text `2/3 cup (55g)`.
 
 ```js
-assert(document.querySelector('span')?.textContent === '2/3 cup (55g)');
+assert(document.querySelector('span')?.textContent.match(/^(\s|)2\/3 cup \(55g\)$/));
```

Comment:
```suggestion
assert(document.querySelector('span')?.textContent?.trim() === '2/3 cup (55g)');
```

Rather than a regex, why not just trim it?

Reply:
yes you are right this is more simple to use trim, thank you for the suggestion